projects/replicated-ship/docs-generated.md

```diff
@@ -1,216 +0,0 @@
-
-## `spec-require-render`
```

Comment:
why is this file getting removed?

Reply:
no idea but it was an effect of running `make project-generated-docs`

Reply:
Huh
I don't get that - when I run `PROJECT=replicated-ship make all` I just have the date change (plus whatever other changes, of course)